docs/CHANGELOG.asciidoc

```diff
@@ -39,6 +40,395 @@ This section summarizes the changes in each release.
 * <<release-notes-7.0.0-alpha1>>
 
 --
+[[release-notes-7.9.0]]
+== {kib} 7.9.0
+coming[7.9.0]
+
+//See also <<breaking-changes-7.9,breaking changes in 7.9>>.
+
+Operations::
+* Moves kibana-keystore from data/ to config/ {kibana-pull}57856[#57856]
+
+[float]
+[[enhancement-7.9.0]]
+=== Enhancements
+Alerting::
```

Comment:
All these items under **Alerting** can move to the **Metrics** as they are specific to metric alerts.